I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/ch6.rpy", line 109, in script
    jump smash
ScriptError: could not find label 'smash'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/ch6.rpy", line 109, in script
    jump smash
  File "renpy/ast.py", line 1757, in execute
    rv = renpy.game.script.lookup(target)
  File "renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'smash'.

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Da Capo 1.0
Mon Jan 10 22:55:09 2022
